i18n/en.json

```diff
@@ -3034,16 +3034,16 @@
   "rhs_card.jump": "Jump",
   "rhs_card.message_by": "Message by {avatar} {user}",
   "rhs_comment.comment": "Comment",
-  "rhs_header.backToFlaggedTooltip": "Back to Flagged Posts",
-  "rhs_header.backToPinnedTooltip": "Back to Pinned Posts",
-  "rhs_header.backToResultsTooltip": "Back to Search Results",
-  "rhs_header.closeSidebarTooltip": "Close Sidebar",
-  "rhs_header.closeTooltip.icon": "Close Sidebar Icon",
+  "rhs_header.backToFlaggedTooltip": "Back to flagged posts",
+  "rhs_header.backToPinnedTooltip": "Back to pinned posts",
+  "rhs_header.backToResultsTooltip": "Back to search results",
+  "rhs_header.closeSidebarTooltip": "Close the sidebar",
+  "rhs_header.closeTooltip.icon": "Close the sidebar icon",
   "rhs_header.details": "Message Details",
-  "rhs_header.expandSidebarTooltip": "Expand Sidebar",
-  "rhs_header.expandSidebarTooltip.icon": "Expand Sidebar Icon",
-  "rhs_header.expandTooltip.icon": "Shrink Sidebar Icon",
-  "rhs_header.shrinkSidebarTooltip": "Shrink Sidebar",
+  "rhs_header.expandSidebarTooltip": "Expand the sidebar",
+  "rhs_header.expandSidebarTooltip.icon": "Expand the sidebar icon",
+  "rhs_header.expandTooltip.icon": "Shrink the sidebar icon",
```

Comment:
Do we need "icon" in the tooltip?

Reply:
Not really, but this acts tooltip acts almost as an alt text for the icon itself, so inserted it to distinguish it from the actual tooltip. Should I remove 'icon'?